dashboards-reports/server/routes/notifications.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { schema } from '@osd/config-schema';
+import { NOTIFICATIONS_DASHBOARDS_API } from '../../common';
+import {
+  IRouter,
+  IOpenSearchDashboardsResponse,
+  ResponseError,
+  Logger,
+  ILegacyScopedClusterClient,
+} from '../../../../src/core/server';
+import { joinRequestParams } from './utils/helpers';
+
+export default function(router: IRouter) {
```

Comment:
Why are we exposing all these API below? If user wants to get configs, they are supposed to go to notificaiton plugin and use API there.

Reply:
So we can eliminate the dependency on dashboards-notifications plugin and only depend on opensearch-notifications. Reporting front-end needs to be able to get available configs to provide the drop-down list of channels when configuring delivery.

Reply:
I thought the dependency on dashboards-notifcation is removed in this PR? According to your PR description. Can we remove these constants then?

Reply:
It is removed in the PR, those constants refer to the reporting endpoints, not the notifications endpoints defined in dashboards-notifications.

Reply:
OK I think I was misled by the constants naming, could you change it to not includes dashboards?
`NOTIFICATIONS_DASHBOARDS_BASE_PATH`

Or, why not just follow the existing pattern in reports.ts for API naming, which uses an `API_PREFIX` from common/index.ts.